src/template.json

```diff
@@ -2104,7 +2104,7 @@
         "modifiers"
       ],
       "abilityMod": {
-        "ability": "cha",
+        "ability": "",
         "mod": 1
```

Comment:
Maybe blank or 0 for mod? I don't THINK it's causing any issues but feels odd to have a default mod if no default ability.

Reply:
I think for almost all (if not all) themes, the default is a bonus of +1, so I want to leave that as the default.